### ISSUE TITLE
Improve repairing of truncated strings

### ISSUE THREAD

**Issue by ceh137** [josdejong/jsonrepair#117]
Hello! 
First of all, thanks a lot for such a great library, really helps me out.

While I was playing with it recently I noticed a nasty bug that can't let me continue working on my project.
The bug is that the comma inside a string is interpreted as a comma in JSON, so that the next word becomes a property of an object.

For you to get a little bit more context, I'm providing an example:
```
{"subject":"Take-Home Assignment Enhancements","text":"Hello Sergey,I hop
```
This string above as an input generates the following:
```
{"subject":"Take-Home Assignment Enhancements","text":"Hello Sergey","I hop":null}
```

Thanks for your help in advance!

**Comment by josdejong** [comment]
Thanks for reporting! Would be nice indeed to improve on this further.

**Comment by josdejong** [comment]
Fixed now in `v3.6.1`.
